zipkin/scribe_writer.py

```diff
@@ -13,14 +13,20 @@ def __init__(self, host, port,  category='zipkin'):
         self.client = Scribe.Client(iprot=protocol, oprot=protocol)
         self.category = category
 
-    def write(self, span):
+    def __span_to_entry(self, span):
```

Comment:
Follows the style that is already there, I would have used a single _ if I had the choice :)